app/src/main/java/com/example/byitscover/scrapers/GoodreadsScraper.java

```diff
@@ -132,9 +132,9 @@ private BigDecimal getPrice(Document bookDocument) {
                 priceString = priceString.substring(0, priceString.length() - 2)
                         + "." + priceString.substring(priceString.length() - 2);
             }
+            return BigDecimal.valueOf(Double.valueOf(priceString));
```

Comment:
To avoid loss of precision, shouldn't this be `new BigDecimal(priceString)`?

Reply:
Yeah I originally thought that I was having an issue doing that so I changed it, but then I think it was a scraper issue and I forgot to go back and change it